src/transformState.js

```diff
@@ -6,6 +6,32 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        for (const key of action.keysToRemove) {
+          if (state[key]) {
+            delete state[key];
+          }
```

Comment:
Remove the check, it can work incorrectly if value is falsy.
Delete works fine with non-existent properties.